webapp-django/crashstats/settings/base.py

```diff
@@ -472,16 +468,16 @@ def filter(self, record):
 SLAVE_DATABASES = config("SLAVE_DATABASES", "", cast=Csv())
 
 
-STATICFILES_FINDERS = (
+STATICFILES_FINDERS = [
     "django.contrib.staticfiles.finders.FileSystemFinder",
     "django.contrib.staticfiles.finders.AppDirectoriesFinder",
     "npm.finders.NpmFinder",
     "pipeline.finders.PipelineFinder",
     # Make sure this comes last!
     "crashstats.crashstats.finders.LeftoverPipelineFinder",
-)
+]
 
-STATICFILES_STORAGE = "pipeline.storage.PipelineCachedStorage"
```

Comment:
Django 3.1 dropped the class that `pipeline.storage.PipelineCacheStorage` subclasses, so we're switching to `PipelineManifestStorage`.